misc/python/materialize/mzcompose.py

```diff
@@ -74,7 +74,7 @@
 )
 
 
-LINT_CONFLUENT_PLATFORM_VERSION = "5.5.3"
+LINT_CONFLUENT_PLATFORM_VERSION = "5.5.4"
```

Comment:
I wish I could also make the linter check the antithesis/cp-combined/Dockerfile but it will require quite a lot of plumbing to get the a single version string specified in a single unified location to be used both in mzcompose.py and in a lint for the Dockerfile